modules/sharedaddy/sharing-service.php

```diff
@@ -504,11 +504,16 @@ function sharing_add_footer() {
 		if ( apply_filters( 'jetpack_sharing_counts', true ) && is_array( $jetpack_sharing_counts ) && count( $jetpack_sharing_counts ) ) :
 			$sharing_post_urls = array_filter( $jetpack_sharing_counts );
 			if ( $sharing_post_urls ) :
+				$is_jetpack = true === apply_filters( 'is_jetpack_site', false, get_current_blog_id() );
```

Comment:
Thanks @jeherve, I wasn't aware of this initiative. I've created a pull request at #2064 (including a few questions to clear up confusion).